addon/src/services/page-title.ts

```diff
@@ -37,24 +64,32 @@ export default class PageTitleService extends Service {
   };
 
   constructor() {
+    // eslint-disable-next-line prefer-rest-params
     super(...arguments);
     this._validateExistingTitleElement();
 
-    let config = getOwner(this).resolveRegistration('config:environment');
+    // @ts-expect-error Property 'resolveRegistration' does not exist on type 'Owner'.ts(2339). Not sure why
+    const config = getOwner(this).resolveRegistration('config:environment') as {
+      pageTitle?: PageTitleConfig;
+    };
+
     if (config.pageTitle) {
-      ['separator', 'prepend', 'replace'].forEach((key) => {
-        if (!isEmpty(config.pageTitle[key])) {
-          this._defaultConfig[key] = config.pageTitle[key];
+      configKeys.forEach((key) => {
+        if (!isEmpty((config.pageTitle as PageTitleConfig)[key])) {
```

Comment:
is it possible to remove all the casting of config.pageTitle?

(or cast once and re-use the variable after asserting that the config matches the expected shape?)

Reply:
<img width="769" alt="Screen Shot 2022-10-15 at 1 04 07 PM" src="https://user-images.githubusercontent.com/34726/195998998-e0309d2c-5f70-4887-b29a-93a23adf7d59.png">
 What I get

Reply:
what's `key`?
and what`s `_defaultConfig`?

Reply:
Both are PageTitleConfig, and key is keyof PageTitleConfig, so 🤷🏼

Reply:
This works https://github.com/ember-cli/ember-page-title/pull/246/commits/eb24065fa1bce2c4fc22b0e3560c2d710b67b6cf

Still doesn't feel great..